tsconfig.json

```diff
@@ -16,5 +16,5 @@
     },
     "esModuleInterop": true
   },
-  "exclude": ["cypress/**", "cdk/**"]
+  "exclude": ["cypress/**", "cdk/**", "cypress.config.ts"]
```

Comment:
https://stackoverflow.com/a/72663546
as suggested by this answer (the jest tests and TS check failed without this exclusion)